db/data/20200907144609_update_organisation_on_all_programme_activities.rb

```diff
@@ -0,0 +1,14 @@
+class UpdateOrganisationOnAllProgrammeActivities < ActiveRecord::Migration[6.0]
+  def up
+    programmes = Activity.programmes
+    beis_organisation_id = Organisation.find_by(service_owner: true).id
+    programmes.each do |programme|
```

Comment:
Is there a reason to loop over these in memory and do an `UPDATE` for each one, rather than using `Activity.programmes.update_all`?

Reply:
You're right there isn't, I was using a pattern from when I updated all the `transparency_id` attributes on Activities before, but for that change we had a different `transparency_id` for each record 🤦

Reply:
That makes sense :)